source/databases-collections.txt

```diff
@@ -0,0 +1,286 @@
+
+=========================
+Databases and Collections
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: table, row, organize, storage, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use MongoDB databases and
+collections with the {+php-library+}.
+
+MongoDB organizes data into a hierarchy of the following levels:
+
+- **Databases**: Top-level data structures in a MongoDB deployment that store collections.
+- **Collections**: Groups of MongoDB documents. They are analogous to tables in relational databases.
+- **Documents**: Units that store literal data such as string, numbers, dates, and other embedded documents.
+  For more information about document field types and structure, see the
+  :manual:`Documents </core/document/>` guide in the {+mdb-server+} manual.
+
+Access a Database
+-----------------
+
+Access a database by passing the database name to the ``MongoDB\Client::selectDatabase()``
+method.
+
+The following example accesses a database named ``test_database``:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.php
+    :language: php
+    :dedent:
+    :start-after: start-access-database
+    :end-before: end-access-database
+
+.. _php-db-coll-access-collection:
+
+Access a Collection
+-------------------
+
+Access a collection by using either of the following methods:
+
+- ``MongoDB\Client::selectCollection()``: Pass the database and collection names as 
+  parameters
+- ``MongoDB\Database::selectCollection()``: Pass the collection name as a parameter
+
+The following example accesses a collection named ``test_collection`` by using the 
+``MongoDB\Database::selectCollection()`` method:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.php
+    :language: php
+    :dedent:
+    :start-after: start-access-collection
+    :end-before: end-access-collection
+
+.. tip::
+
+   If the provided collection name does not already exist in the database,
+   MongoDB implicitly creates the collection when you first insert data
+   into it.
+
+.. _php-db-coll-create-collection:
+
+Create a Collection
+-------------------
+
+Pass a collection name to the ``MongoDB\Database::createCollection()`` method to
+explicitly create a collection in a MongoDB database.
+
+The following example creates a collection named ``example_collection``:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.php
+    :language: php
+    :dedent:
+    :start-after: start-create-collection
+    :end-before: end-create-collection
+
+You can specify collection options, such as maximum size and document
+validation rules, by passing them as an array to the ``createCollection()`` method.
+For a full list of optional parameters, see the `API documentation
+<{+api+}/method/MongoDBDatabase-createCollection/#parameters>`__.
```

Comment:
I assume `:phpmethod: is not used here because you want to deep link to the parameters heading?

Reply:
yes!